src/components/MyProfile.jsx

```diff
@@ -18,4 +18,26 @@ function MyProfile() {
   );
 }
 
+const Rocket = () => {
+  const Rocket = useSelector((state) => state.rockets);
+  const myMission = Rocket.RocketList.filter((item) => item.reserved === true);
+  if (myMission.length === 0) return (<Container>You have not reserved any Rocket.</Container>);
+  return (
+    <Container>
+      <ListGroup as="ol" numbered>
+        {myMission.map((item) => (
+          <ListGroup.Item as="li" key={item.id}>{item.name}</ListGroup.Item>
+        ))}
+      </ListGroup>
+    </Container>
+  );
+};
+
+const MyProfile = () => (
+  <>
+    <Mission />
+    <Rocket />
+  </>
```

Comment:
###[OPTIONAL] Good job so far feel free to fix this small issue check this image the mission and rocket should be side by side you can add my suggestions to your code it will fix it.
![image](https://github.com/Elhambasir/group-project-space-travelers-hub/assets/108110651/b20adb0a-7427-4fc4-a10c-036bafb1b97e)

``` suggestion
  <>
<Container className="d-flex">
      <Mission />
      <Rocket />
    </Container>
  </>
```